Add copy edits based on suggestions (#545)
Consensys/doc.teku: docs/concepts/withdrawals.md

Validators staking ether on Mainnet after [The Merge](merge.md), accrue two
forms of rewards:

> - execution layer rewards paid directly to a withdrawal address (Ethereum address)

Reviewer: Are you sure this line and the next should not start with the first word in initial UC?

Author: https://www.microsoft.com/en-us/microsoft-365-life-hacks/writing/punctuating-bullet-point-lists#:~:text=Be%20consistent,are%20following%20the%20same%20format.

Author: Sorry @mjsmike62 you were 100% correct!

> - consensus layer rewards for performing actions each epoch.

Author: I think since they're fragments continuing from the introductory clause, they start with lowercase letters and are separated by commas, with a period at the end of the last bullet.

Reviewer 2: Please review the list guidelines from the Microsoft Style Guide: https://learn.microsoft.com/en-us/style-guide/scannable-content/lists. Items should always start with an uppercase letter, even if they're fragments, unless they start with a technical term that is always lowercase.

From the first Capella slot, block proposers provide up to 16 withdrawals per
proposed block.

> Proposers start with validator 1, and find validators that have an Ethereum address as a withdrawal address, and meet one of these criteria:

Reviewer: Maybe...

Starting with validator 1, proposers find validators that have an Ethereum address as a withdrawal address and have done one of the following:

- Excess balance
- Exited <something?>

> - have an Ethereum address as a withdrawal address,
> - posses an excess balance,
> - have exited.

Reviewer 2: ```suggestion
- have an Ethereum address as a withdrawal address,
- posses an excess balance,
- have exited.
```

Please review the list guidelines from the Microsoft Style Guide: https://learn.microsoft.com/en-us/style-guide/scannable-content/lists. Items should start with an uppercase letter and end with the same punctuation.

> Proposers start with validator 1 and search for validators that meet any of these criteria:
> - have an Ethereum address as a withdrawal address,
> - posses an excess balance,
> - have exited.

Reviewer 2: ```suggestion
Proposers start with validator 1, and find validators that have an Ethereum address as a withdrawal address, and meet one of these criteria:

- They have an excess balance.
- They have exited.
```

If you're unsure about the meaning, first read the context in the page to try to understand (that's what I did here to extract this list).

Reviewer 2: Also, please don't use synonyms when consistent, simple words suffice (i.e., "have" and "possess").

Reviewer 2: @joaniefromtheblock can you take a look at these comments?

From the first Capella slot, block proposers provide up to 16 withdrawals per
proposed block.

> Proposers start with validator 1 and search for validators that meet any of these criteria: they have an Ethereum address as a withdrawal address, possess an excess balance, or have exited.

Reviewer: Maybe... 

Proposers start with validator 1 and locate validators that meet any of these criteria: 
- An Ethereum address as a withdrawal address
- An excess balance
- Have exited

These edits are my impressions w/o actually using the product.
